### ISSUE TITLE
Sobre SEO

### ISSUE THREAD

**Issue by pedrojovelli** [robanquieri/rioslumstudio#55]
Se souber responder...

é possível mostrar mais informações sobre a empresa quando somos listados no Google?

<img width="906" alt="screen shot 2017-12-19 at 7 53 28 am" src="https://user-images.githubusercontent.com/15367213/34158256-ca00aade-e4ab-11e7-9c2c-7170407e4580.png">

**Comment by robanquieri** [comment]
Sim, mas não sei quanto tempo leva, me passa um titulo da empresa e uma descrição.

**Comment by pedrojovelli** [comment]
ok. Vou checar isso.
